tools/shell-commands/src/main/resources/requestBody/2.4.0/base_update_by_query_request.json

```diff
@@ -0,0 +1,22 @@
+{
+  "script": {
+    "source": "#painless",
+    "lang": "painless"
+  },
+  "query": {
```

Comment:
Could we identify the items to update directly from the query ?
meaning the check are still good in the painless, but if the query is able to return less items to be updated by the painless it will be probably faster to execute.

being able to query only document with: `target.properties.path` that exists